### PR TITLE
change Callable[..., bool] to Callable[..., Any]

### PR DESCRIPTION
**Pull request by ernestoarbitrio**
here the reason: https://docs.python.org/3/library/operator.html#operator.lt

### CODE REVIEW

File: src/histolab/util.py [hunk 1]
```diff
@@ -206,6 +204,7 @@ def threshold_to_mask(
     np.ndarray
         Boolean NumPy array representing a mask where a pixel has a value True
         if the corresponding input array pixel exceeds the threshold value.
+        if the corresponding input array pixel exceeds the threshold value.
```
Comment (alessiamarcolini) [inline src/histolab/util.py:207]: ?

**Review by alessiamarcolini — COMMENTED** [review]
I'm not really understanding why 😇

**Comment by ernestoarbitrio** [comment]
It's formally wrong . So if in future we wanna integrate a static type checker like mypy this will be marked as an error. Also my ide says that the return type is wrong.

File: src/histolab/util.py (hunk 1 above)
Comment (ernestoarbitrio) [reply src/histolab/util.py:207]: there was a blank line, i don't know why github is marking this change this way

**Comment by ernestoarbitrio** [comment]
> I'm not really understanding why this change is needed 😇

left a general comment, furthermore if you read here  https://docs.python.org/3/library/operator.html#operator.lt there is this sentence that makes the things clear:
```
Note that these functions can return any value, which may or may not be interpretable as a Boolean
```

**Comment by alessiamarcolini** [comment]
Oh, now I get it, I probably overlooked that sentence

**Review by alessiamarcolini — APPROVED** [review]

**Pull request merged**